app/models/object_store/item.rb

```diff
@@ -0,0 +1,33 @@
+module ObjectStore
+  class Item < Edition
+    store_accessor :details, :item_type
+
+    after_initialize :add_field_accessors
+
+    def summary_required?
+      false
+    end
+
+    def body_required?
+      false
+    end
+
+    def previously_published
+      false
+    end
+
+  private
+
+    def add_field_accessors
```

Comment:
If we were to reach for a field object we could decouple ourselves from ObjectStore a little with something like:

```ruby
def fields
  @fields ||= ObjectStore.fields_for_item_type(item_type)
end

def add_field_accessors
  @fields.each do |field|
    singleton_class.class_eval { store_accessor :details, field.name }
    if field.required?
      singleton_class.class_eval { validates fields.name, presence: true }
    end
  end
rescue UnknownItemType
  # Ignored
end
```